.gitignore

```diff
@@ -32,12 +32,12 @@
 
 /examples/private_xo/target
 /examples/private_xo/Cargo.lock
-/examples/private_xo/demo_state/keys.yaml
```

Comment:
typo in commit message: splitnerd -> splinterd

Reply:
Fixed